Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -783,19 +783,19 @@ public void RaiseFocusExit(IMixedRealityPointer pointer, GameObject unfocusedObj
             };
 
         /// <inheritdoc />
-        public void RaisePointerDown(IMixedRealityPointer pointer, MixedRealityInputAction inputAction)
+        public void RaisePointerDown(IMixedRealityPointer pointer, Handedness handedness, MixedRealityInputAction inputAction)
```

Comment:
pointers likely shouldn't know about handedness, as they're not controllers, but objects controlled and driven by controllers.

Likely the InputDown/up should be used instead.

Reply:
Pointer events [already know about handedness](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3294/files/725d665e1f3a8440b737bea2ad77cf4b68b751f4#diff-83ff2790c22f67c0d7d4a7a413be9c29L795). This isn't anything new, but git is confusing the two lines. My link above links to this exact line already existing.

Reply:
Maybe that's the problem. They shouldn't. The input source is what should know about handedness.

Reply:
The problem with what?

Reply:
The pointer doesn't have a handedness property. So the property shouldn't be passed through the event data.

The pointer should only have a reference to the input source, who's controller implementation has the handedness property.

Again. Pointers shouldn't know about handedness.

Reply:
I'm not adding anything new here. The existing methods and events all have handedness information. If you'd like to propose removing them, please file a proposal for discussion. This PR has nothing to do with handedness.

Reply:
Well then let's come back to this PR once that change is in.

Give me 10 min.

Reply:
This PR doesn't depend on a change to handedness. That proposal can be addressed following this one.

Reply:
Yes it does, because I don't want to make a change to the IMixedRealityInputSystem contract more than once.